plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -76,6 +76,7 @@ export type StepperProps = {
   manifest: TemplateParameterSchema;
   extensions: FieldExtensionOptions<any, any>[];
   templateName?: string;
+  minutesSaved?: number;
```

Comment:
It's a little unfortunate to add this to the `Stepper` props.

Is there a better way that we can refactor this instead rather than leaking out this detail into the props?

Wondering if we can move the tracking entirely to a parent component? Or provide something more generic like `trackingContext` or something as a prop?

Reply:
Yeah lemme see what I can do.

Reply:
> _Wondering if we can move the tracking entirely to a parent component?_

So my worry with moving the tracking out of the stepper itself is that any custom usages of the stepper (outside the context of `<Workflow>`, for example) would suddenly lose tracking.  ...Which isn't ideal.

Happy to add a `trackingContext` prop (and deprecate `templateName`, which is only used in the tracking code) though.

Reply:
Yeah something like that, If you can find a better name also please go ahead, but thinking it's a little cleaner than adding more props for now.

:tada:

Reply:
Per discussion offline: given the `<Stepper>` probably shouldn't be used outside the context of a `<Workflow>` anyway, it makes sense to lift the event capture up there and then not have to deal with passing props down at all.

Doing so may even fix a bug where the `create` event was possibly being fired in the template editor form.

Reply:
Nice!